astropy/coordinates/representation/base.py

```diff
@@ -818,7 +818,7 @@ def _re_represent_differentials(self, new_rep, differential_class):
         ):
             # TODO: is there a better way to do this?
```

Comment:
It looks like with the changes to the code the comment will be outdated.
```suggestion
```

Reply:
✅